src/components/Widget/index.js

```diff
@@ -325,6 +326,14 @@ class Widget extends Component {
     }
   }
 
+  checkVersionBeforePull() {
+    const { storage } = this.props;
+    const localSession = getLocalSession(storage, SESSION_NAME);
+    if (localSession && (localSession.version !== 'packageVersionToBeReplaced')) {
```

Comment:
Since it's kind of a constant, I think this would look better in all caps  PACKAGE_VERSION_TO_BE_REPLACED